src/field_10x26.h

```diff
@@ -9,15 +9,22 @@
 
 #include <stdint.h>
 
+/** Implements arithmetic modulo FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFE FFFFFC2F,
+ *  represented as 10 uint64_t's in base 2^26, least significant first. Note that the limbs are allowed to
+ *  contain >26 bits each. */
 typedef struct {
     /* X = sum(i=0..9, n[i]*2^(i*26)) mod p
      * where p = 2^256 - 0x1000003D1
      */
     uint32_t n[10];
-#ifdef VERIFY
-    int magnitude;
-    int normalized;
-#endif
+    /*
+     * Magnitude m implies n[i] <= 2*m*0x3FFFFFF for i=0..8, and
+     * n[9] <= 2*m*0x3FFFFF (a factor 16 smaller).
+     *
+     * Normalized requires magnitude 1, all n[i] <= 0x3FFFFFF, and
```

Comment:
"requires magnitude <= 1"?

Reply:
This has been changed significantly; the rule that normalized implies magnitude 0 or 1 has now been added to field.h itself (it's in fact a rule that must hold for all users of field.h, independent of which field implementation was used).